### ISSUE TITLE
results = model.predict() from yolov8 equivalent object

### ISSUE THREAD

**Issue by CodeByHarri** [WongKinYiu/yolov9#208]
im looking to get the results object equivalent from yolov8 but in yolov9.

yolov8:
```python
results = model.predict(image, save=True, iou=0.3, save_txt=True, conf=0.06)

for r in results:
  bounding_boxes = r.boxes.xyxy.numpy()
```

currently im trying detect.py but my results .txt files bounding boxes are between 0 and 1, unlike my yolov8 model which gives me values that are between 0-512 which is my image height/width.

yolov9:
```python
!python detect.py \
--img 512 --conf 0.1 --iou 0.7 --device 'cpu' --save-txt --save-conf --line-thickness 1\
--weights /gdrive/MyDrive/Datasets/070324models/train/exp/weights/best.pt \
--source '/content/Submission data'
```

how do i get the detect.py to give me results like yolov8 where the bounding boxes are between my image width/height

**Comment by CodeByHarri** [comment]
https://github.com/WongKinYiu/yolov9/issues/186#issuecomment-1981055108

I've tried using this function, but the bounding boxes do not give me the same results as the yolov8 method.  :/

**Comment by shoukewei** [comment]
It would be great to easily get the bbox and its center point.

**Comment by CodeByHarri** [comment]
this edit to the detect.py file makes it work:

```python
                for *xyxy, conf, cls in reversed(det):
                    if save_txt:  # Write to file
                        print(torch.tensor(xyxy).view(1, 4))
                        xywh = (torch.tensor(xyxy).view(1, 4)).view(-1).tolist()  # normalized xywh
                        line = (cls, conf, *xywh) if save_conf else (cls, *xywh)  # label format
                        with open(f'{txt_path}.txt', 'a') as f:
                            f.write(('%g ' * len(line)).rstrip() % line + '\n')
```


getting rid of the /gn because that normalises the values, also switching the file around so it shows class, conf and then xyxy.
